apps/client/src/shared/page-header/page-header.component.html

```diff
@@ -24,7 +24,7 @@
         </div>
         
         <div class="d-flex justify-content-end align-items-center">
-            <a href="/logout" (click)="onLogout()" class="btn btn-link" title="Logout">
+            <a href="/logout" (click)="onLogout($event)" class="btn btn-link" title="Logout">
```

Comment:
```suggestion
            <a href="/logout" (click)="onLogout" class="btn btn-link" title="Logout">
```
I think this would have had the same effect